libmscore/repeatlist.cpp

```diff
@@ -415,7 +425,7 @@ void RepeatList::unwindSection(Measure* sectionStartMeasure, Measure* sectionEnd
                   }
             if (doJump && !isGoto) {
                   Jump* s = 0;
-                  foreach(Element* e, m->el()) {
+                  foreach (Element* e, m->el()) {
```

Comment:
Seems like there are quite a lot of "foreach" in the repo: https://github.com/musescore/MuseScore/search?utf8=%E2%9C%93&q=foreach